doc_build/BUILD

```diff
@@ -26,6 +28,7 @@ load("//:version.bzl", "version")
 # buildifier: leave-alone
 ORDER = [
     # buildifier: do not sort
+    # buildifier: do not reformat
```

Comment:
No. It is not. But I am hoping to slip it in. I may need my own fork of buildifier for that. 
I imported 0.6.0 into Google the other day and spent too long figuring out a problem with this mostly because the mandatory buildifier on import ruined the indentation as comments this provides.